hyperspy_swift_library/__init__.py

```diff
@@ -69,11 +69,22 @@ def __init__(self, workspace_dir):
             di.read_properties() for di in self._data_items]
 
     def get_data_items(self):
-        """ Returns DataFrame if Pandas is installed containing the data_items properties in a Nion Library
-        Returns _data_items_properties is panda is not installed
+        """	
+        Parameters
```

Comment:
I think that this method doesn't take any parameters

Reply:
You are right. I will modify that.